app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -5,7 +5,7 @@ import { NO_REMINDER_ICON } from "discourse/models/bookmark";
 import UserMenuTab, { CUSTOM_TABS_CLASSES } from "discourse/lib/user-menu/tab";
 import { inject as service } from "@ember/service";
 import getUrl from "discourse-common/lib/get-url";
-import DiscourseURL from "discourse/lib/url";
+import { wantsNewWindow } from "discourse/lib/intercept-click";
```

Comment:
This is amazing, I didn't know about this helper.

Reply:
Yeah it's super useful - saves having to manually check all the different key combinations everywhere.